src/main/scala/org/broadinstitute/dsde/workbench/leonardo/db/ClusterComponent.scala

```diff
@@ -0,0 +1,93 @@
+package org.broadinstitute.dsde.workbench.leonardo.db
+
+import java.sql.Timestamp
+import java.util.UUID
+
+import org.broadinstitute.dsde.workbench.leonardo.model.{Cluster, ClusterStatus}
+
+case class ClusterRecord(clusterId: UUID,
+                         clusterName: String,
+                         googleProject: String,
+                         googleServiceAccount: String,
+                         googleBucket: String,
+                         operationName: String,
+                         status: String,
+                         createdDate: Timestamp,
+                         destroyedDate: Option[Timestamp])
+
+trait ClusterComponent extends LeoComponent {
+  this: LabelComponent =>
+
+  import profile.api._
+
+  class ClusterTable(tag: Tag) extends Table[ClusterRecord](tag, "CLUSTER") {
+    def clusterId =             column[UUID]              ("clusterId",             O.PrimaryKey)
```

Comment:
Confused - so there's should be 2 ids right? - google's cluster Id and our own internal Id